network/p2p/p2p.go

```diff
@@ -178,6 +182,12 @@ func MakeService(ctx context.Context, log logging.Logger, cfg config.Local, h ho
 	h.SetStreamHandler(AlgorandWsProtocol, sm.streamHandler)
 	h.SetStreamHandler(libp2phttp.ProtocolIDForMultistreamSelect, sm.streamHandlerHTTP)
 
+	// set an empty handler for telemetryID/telemetryInstance protocol in order to allow other peers to know our telemetryID
+	telemetryID := log.GetTelemetryGUID()
+	telemetryInstance := log.GetInstanceName()
```

Comment:
This field is a base64 string with the standard alphabet, so it could contain `/`. That would not pass the parsing function

Also it seems `GetTelemetryGUID` can return a string which contains `TelemetryConfig.Name`, which can have any set of characters in it, so similar problem there

Reply:
good point, I forgot base64 has slash in its alphabet. I'll base32 encoding there

Reply:
done